link_shortener/models.py

```diff
@@ -13,6 +13,7 @@
     Column('identifier', String(36)),
     Column('owner', String(50)),
     Column('owner_id', String(255)),
+    Column('password', String(30), default=''),
```

Comment:
I think it might be more obvious to use NULL in case of no password but it's up to you...

Reply:
Changed